### PR TITLE
II-8345: Update selenium integration documentation

### PR DESCRIPTION
**Pull request by inga-balcune**
Resolves #
@Jym77 I briefly reviewed the documentation and updated what I found, but please let me know if I missed some parts.

### CODE REVIEW

File: packages/alfa-test-utils/docs/usage/scraping/integrated.md [hunk 1]
```diff
@@ -78,29 +78,23 @@ Most of this code is standard in test automation and likely to exist in end-to-e
 
 ## Selenium
 
-The Selenium integration is not fully built, so it requires a bit more manual steps to scrape the page.
-
-Install the needed packages:
+Install the `@siteimprove/alfa-playwright` package:
```
Comment (Jym77) [inline packages/alfa-test-utils/docs/usage/scraping/integrated.md:81]:
```suggestion
Install the `@siteimprove/alfa-selenium` package:
```

File: packages/alfa-test-utils/docs/usage/scraping/integrated.md [hunk 2]
```diff
@@ -78,29 +78,23 @@ Most of this code is standard in test automation and likely to exist in end-to-e
...
 
 ```shell
-$ npm install --save-dev @siteimprove/alfa-device @siteimprove/alfa-dom @siteimprove/alfa-http @siteimprove/alfa-web
+$ npm install --save-dev @siteimprove/alfa-playwright
```
Comment (Jym77) [inline packages/alfa-test-utils/docs/usage/scraping/integrated.md:84]:
```suggestion
$ npm install --save-dev @siteimprove/alfa-selenium
```

File: packages/alfa-test-utils/docs/usage/scraping/integrated.md [hunk 3]
```diff
@@ -78,29 +78,23 @@ Most of this code is standard in test automation and likely to exist in end-to-e
...
 ```
 
 or
 
 ```shell
-$ yarn add --dev @siteimprove/alfa-device @siteimprove/alfa-dom @siteimprove/alfa-http @siteimprove/alfa-web
+$ yarn add --dev @siteimprove/alfa-playwright
```
Comment (Jym77) [inline packages/alfa-test-utils/docs/usage/scraping/integrated.md:90]:
```suggestion
$ yarn add --dev @siteimprove/alfa-selenium
```

**Review by Jym77 — CHANGES_REQUESTED** [review]
Copy/paste eagerness :-D 

Can you also remove the last sentence of https://github.com/Siteimprove/alfa-integrations/blob/main/packages/alfa-test-utils/docs/usage/scraping/advanced.md ?

**Comment by Jym77** [comment]
🙈 The failed actions are on me 🤦

**Comment by inga-balcune** [comment]
> actions are on m

Oops, and bad copy-paste job! It's getting late 🥱

**Review by Jym77 — CHANGES_REQUESTED** [review]
Can you also remove the last sentence of https://github.com/Siteimprove/alfa-integrations/blob/main/packages/alfa-test-utils/docs/usage/scraping/advanced.md ? 
(same reference to Selenium as in the Alfa hub)

**Review by Jym77 — APPROVED** [review]

**Pull request merged**